src/main/java/org/opentripplanner/transit/model/timetable/RealTimeState.java

```diff
@@ -31,4 +31,9 @@ public enum RealTimeState {
    * trip pattern of the scheduled trip.
    */
   MODIFIED,
+
+  /**
+   * The trip has been canceled, and should not be visible to the end user
```

Comment:
This is not true. It has been replaced, not cancelled.

Reply:
Well, in GTFS language it is deleted. This means that you should never ever show it to a user.

Reply:
This definition comes from the GTFS-RT specification, which we use as a base

Reply:
Updated description